internal/provider/incident_catalog_entries_resource.go

```diff
@@ -399,7 +399,7 @@ func (r *IncidentCatalogEntriesResource) getEntries(ctx context.Context, catalog
 	for {
 		result, err := r.client.CatalogV2ListEntriesWithResponse(ctx, &client.CatalogV2ListEntriesParams{
 			CatalogTypeId: catalogTypeID,
-			PageSize:      lo.ToPtr(int(250)),
+			PageSize:      lo.ToPtr(int64(250)),
```

Comment:
Just for my own curiosity, what prompted this?

Reply:
It's because we've updated our openapi schema. My guess is that we've changed the type in our own API definition, which is why the openapi generator has reflected it here